config/jobs/kubernetes/sig-testing/ec2-e2e.yaml

```diff
@@ -11,8 +11,9 @@ presubmits:
       preset-e2e-containerd-ec2: "true"
       preset-dind-enabled: "true"
     path_alias: k8s.io/kubernetes
-    always_run: false
+    always_run: true
```

Comment:
can we please add a description or a better name? "GCE" is understood to be the shell scripts, with newer tools we've included something clearer referencing the bootstrapping tool "kops" or "kind". or if this is node e2es and not cluster e2es we can add "node" to the name.

Reply:
This uses `kubetest2-ec2`[1]  to stand up the cluster. I'll try to make it clearer in the description.

[1] https://github.com/kubernetes-sigs/provider-aws-test-infra/tree/main/kubetest2-ec2#readme

Reply:
this is not a node e2e job. This is the equivalent for `pull-kubernetes-e2e-gce` for the record.

Reply:
I see. I still think the naming is vague and contributors etc probably won't know what this means.